### PR TITLE
[Agent] Send hostname when enrolling

### PR DESCRIPTION
**Pull request by michalpristas**
Fix enroll to include os.Hostname when enroll is performed

Fixes: #15465

### CODE REVIEW

File: x-pack/agent/pkg/agent/application/enroll_cmd.go [hunk 1]
```diff
@@ -165,11 +171,17 @@ func (c *EnrollCmd) Execute() error {
...
+	hostname, err := os.Hostname()
+	if err != nil {
+		return nil, err
+	}
+
 	return map[string]interface{}{
 		"platform": runtime.GOOS,
 		"version":  release.Version(),
-	}
+		"host":     hostname,
```
Comment (ruflin) [inline x-pack/agent/pkg/agent/application/enroll_cmd.go]: Would be good to follow ECS also for this kind of information.

Comment (michalpristas) [reply x-pack/agent/pkg/agent/application/enroll_cmd.go]: ^^ @nchaulet

Comment (ruflin) [reply x-pack/agent/pkg/agent/application/enroll_cmd.go]: Please also take https://www.elastic.co/guide/en/ecs/current/ecs-os.html into account.

Comment (ph) [reply x-pack/agent/pkg/agent/application/enroll_cmd.go]: Do you mean sending the data already as an ECS formatted document if so  I am OK with this proposal.

**Comment by nchaulet** [comment]
@ph yes it’s not validated

**Comment by michalpristas** [comment]
I created issue #15500 for our side of story

**Pull request merged**